src/pages/home/report/ReportActionCompose.js

```diff
@@ -175,10 +174,6 @@ class ReportActionCompose extends React.Component {
     }
 
     render() {
-        // We want to make sure to disable on small screens because in iOS safari the keyboard up/down buttons will
-        // focus this from the chat switcher.
-        // https://github.com/Expensify/Expensify.cash/issues/1228
-        const inputDisable = this.props.isSmallScreenWidth && Navigation.isDrawerOpen();
```

Comment:
This check is no longer needed as you would see below the ReportActionCompose is unmounted when the drawer is expanded